libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/atoms/UnreadIndicatorAtom.kt

```diff
@@ -37,12 +37,13 @@ fun UnreadIndicatorAtom(
     modifier: Modifier = Modifier,
     size: Dp = 12.dp,
     color: Color = ElementTheme.colors.unreadIndicator,
+    isVisible: Boolean = true,
 ) {
     Box(
         modifier = modifier
             .size(size)
             .clip(CircleShape)
-            .background(color)
+            .background(if (isVisible) color else Color.Transparent)
```

Comment:
Why not use an if to decide if the indicator should be displayed at all? Do we need to have the view taking space for some layout reasons?

Reply:
Yes, I did not want to change the behavior here, but I agree we could just remove the indicator. Will revisit later.